lib/pbench/test/unit/server/conftest.py

```diff
@@ -323,6 +330,49 @@ def attach_dataset(create_drb_user, create_user):
         ).add()
 
 
+@pytest.fixture()
+def more_datasets(
+    client, server_config, create_drb_user, create_admin_user, attach_dataset
+):
+    """
+    Supplement the conftest.py "attach_dataset" fixture with a few more
+    datasets so we can practice various queries. In combination with
+    attach_dataset, the resulting datasets are:
+
+        Owner   Access  Date        Name
+        ------- ------- ----------- ---------
+        drb     private 2020-02-15  drb
+        test    private 2002-05-16  test
+        drb     public  2020-02-15  fio_1
+        test    public  2002-05-16  fio_2
+
+    Args:
+        client: Provide a Flask API client
+        server_config: Provide a Pbench server configuration
+        create_drb_user: Create the "drb" user
+        create_admin_user: Create the "test_admin" user
+        attach_dataset: Provide some datasets
```

Comment:
I don't think we need to reference `create_drb_user` here, because I think we "inherit" it via `attach_dataset` (and we don't reference that fixture explicitly in this function, just its side-effect).

Likewise, I don't think we need to reference `create_admin_user`, since we don't reference it (or its user) in this function...unless you had intended to....  👀  (We reference the `"test"` `owner`, which I think we get via the inherited `create_user` fixture.)

Reply:
I only moved this fixture, and I didn't really feel the need to re-evaluate the dependencies. You may be right, but I'm not sure it's relevant to this PR.

Reply:
The question of relevance is hard to evaluate:  it appears to be new code (but, as you say, it was deleted elsewhere and reinstantiated here), which makes this a rather philosophical question.

Obviously, the code functions as is.  Mostly, I was trying to make sure that you hadn't mistyped something or left something out which was supposed to make use of `create_admin_user`.  If that's not the case, then we're good here (unless you _want_ to remove the unneeded fixture references...which I would encourage 🙂).